src/stores/connectors/ethereum/utils.go

```diff
@@ -27,7 +25,7 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 }
 
 func getEIP191EncodedData(msg []byte) string {
```

Comment:
Minor: This method should return the []byte directly as returned values is always used as []byte not as string